src/alpaka/bin/main.cc

```diff
@@ -123,7 +148,9 @@ int main(int argc, char** argv) {
             << numberOfThreads << " threads." << std::endl;
 
   // Initialize tasks scheduler (thread pool)
-  tbb::task_scheduler_init tsi(numberOfThreads);
+  // tbb::task_scheduler_init tsi(numberOfThreads);
+  // TO DO: Warning: this will not work, need to be included in header.
+  // See external/alpaka/include/alpaka/kernel/TaskKernelCpuTbbBlocks.hpp
```

Comment:
I took a quick look of this file and it is not clear to me what the problem is.

Reply:
I could not change the number of threads in the TBB pool by using `tbb::task_scheduler_init tsi(numberOfThreads);` in main (had no effect).
Instead, by adding a `tbb::task_scheduler_init tsi(numberOfThreads);` in external/alpaka/include/alpaka/kernel/TaskKernelCpuTbbBlocks.hpp , I could control the number of threads in the TBB pool.

Reply:
Thanks for the explanation. This needs to be understood better (because in CMSSW we must have a `task_scheduler_init` , well `global_control` nodaways, done in `main()` and tasks from Alpaka should go into the same thread pool), but can be done after this PR.